build-tools/scripts/omnisharp.json.in

```diff
@@ -0,0 +1,20 @@
+{
+    "msbuild": {
+        "loadProjectsOnDemand": true,
+    },
+
+    "RenameOptions": {
+        "RenameInComments": false,
+        "RenameOverloads": true,
+        "RenameInStrings": false
+    },
+
+    "sdk": {
+        "IncludePreleases": true,
+        "Path": "@DOTNET_SDK_PATH@",
+    },
+
+	"msbuild": {
```

Comment:
I get an error about a duplicate configuration here, this section can probably be merged with the one at the top of the file.

Reply:
Seriously, can JSON be any worse...?